### DISCUSSION TITLE
Using asyncio for notion.pages.create requests

### DISCUSSION THREAD

**Discussion by JonathanBonnaud** [ramnes/notion-sdk-py#206] [Q&A] [answered]
Hello,

Is it possible to use `notion.pages.create` in an async way? (or any other `notion.pages.update`, `notion.pages.retrieve`)
I tried, but not with a lot of success.

Thanks :)

**Comment by ramnes** [comment, marked as answer]
Hey there!

Yes, all methods are available as coroutines when you use `AsyncClient`, as written in the README. :)

For example:

```python
import asyncio
import os
from pprint import pprint

from notion_client import AsyncClient


async def main():
    notion = AsyncClient(auth=os.environ["NOTION_TOKEN"])
    users = await notion.users.list()
    pprint(users)


asyncio.run(main())
```
